.github/workflows/pypi_upload.yml

```diff
@@ -14,6 +14,8 @@ jobs:
 
     - name: Set up Python
       uses: actions/setup-python@v4
+      with:
+        python-version: 3.x
```

Comment:
Doesn't this default to that or did I break it? Happy to be more explicit here too.

Reply:
It used to be REQUIRED but perhaps they _fixed_ it after user complaints.  If your mods work then it is now optional.

Reply:
https://github.com/pypa/bandersnatch/blob/main/.github/workflows/pypi_upload.yml#L18 - Ahh seems I did * here to get latest available python 3

Reply:
That would actually autoupgrade to Python 4!!!  You _are_ brave.  ;-)

Reply:
Ya, I roll forward and just fix when things break everywhere ... rather than pin and forget to move things like most people seem to prefer to do which I don't get ...